www/js/viewer.js

```diff
@@ -218,7 +218,8 @@ const createCards = (rows, LineID) => {
   Object.keys(rows).forEach((key) => {
     row = rows[key];
     lines.push(row.ID);
-    const gurmukhiShabads = row.GurmukhiBisram.split(' ');
+    const taggedGurmukhi = row.Gurmukhi.split(' ');
```

Comment:
Are there any `]` characters in this column? In which case we would need that loop for the first condition around line 224?

Reply:
Updated to just revert the changes from here: https://github.com/KhalisFoundation/sttm-desktop/commit/31bac4747c85b05669e80017e61e96f2c88458c2#diff-df6fefbebc662cf1cd3f5236a8ae670d